include/aligator/modelling/centroidal/centroidal-wrench-cone.hpp

```diff
@@ -52,17 +52,20 @@ struct WrenchConeResidualTpl : StageFunctionTpl<_Scalar> {
 };
 
 template <typename Scalar>
-struct WrenchConeDataTpl : StageFunctionDataTpl<Scalar> {
+struct CentroidalWrenchConeDataTpl : StageFunctionDataTpl<Scalar> {
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
   using Base = StageFunctionDataTpl<Scalar>;
 
   Eigen::Matrix<Scalar, 17, 6> Jtemp_;
 
-  WrenchConeDataTpl(const WrenchConeResidualTpl<Scalar> *model);
+  CentroidalWrenchConeDataTpl(
+      const CentroidalWrenchConeResidualTpl<Scalar> *model);
 };
 
 } // namespace aligator
 
+#include "aligator/modelling/centroidal/centroidal-wrench-cone.hxx"
```

Comment:
No need to include the implementation hxx file here, just in the corresponding cpp file.
```suggestion
```